documentation/16-start-stop/README.md

```diff
@@ -0,0 +1,27 @@
+Title: preStart, preStop, & postStop
```

Comment:
I tried it on the lifecycle page, but it felt buried there. This is significant enough that I think it deserves to stand on its own.